lib/core/api.js

```diff
@@ -84,6 +85,40 @@ api.user.logout = function(cb) {
   tidepool.logout(cb);
 };
 
+api.user.getUploadGroups = function(cb) {
+  var userId = tidepool.getUserId();
+
+  api.log('GET /access/groups/' + userId);
+
+  tidepool.getViewableUsers(userId, function(err, groups) {
+    if (err) {
+      return cb(err);
+    }
+
+    common.step([
```

Comment:
@ianjorgensen you introduced common here - you should be using the already used async.